### PR TITLE
Add network utils

### PR DESCRIPTION
**Pull request by coryschwartz**
This is largely taken from the bitswap tuning, and made a little more general

Does it make sense to come up with some common/well-known toml-configuration params?

### CODE REVIEW

File: sdk/utils/network.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package utils
```
Comment (nonsense) [inline sdk/utils/network.go]: nitpick: maybe we could name this package `network`, with methods called `Setup` and `SetupWaitAll`, then we avoid having a generic name for a package.

Comment (Stebalien) [reply sdk/utils/network.go]: Agree. If we do that, we should move all network helpers here (e.g., `sync.WaitNetworkInitialized`).

Comment (coryschwartz) [reply sdk/utils/network.go]: Sounds good, I will do that.

File: sdk/sync/helper.go [hunk 2]
```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/ipfs/testground/sdk/runtime"
 )
 
+// Depreciated. Use sdk.network.WaitNetworkInitialized
```
Comment (Stebalien) [inline sdk/sync/helper.go]: This is going to cause the lints to fail. We should do a mass update.

**Comment by Stebalien** [comment]
LGTM once tests pass.

**Comment by coryschwartz** [comment]
I'm closing this for now. I think there will be a more comprehensive plan SDK review in the future.

**Comment by Robmat05** [comment]
Ok, cool. We will close this as ‘Discarded’ and look for it to be linked to a new ticket on SDK review, once it is opened.

**Pull request closed without merging**